slides/developer-guide/working-with-shapes/working-with-word-art/_index.md

```diff
@@ -134,7 +134,7 @@ curl -X POST "https://api.aspose.cloud/v3.0/slides/MyPresentation.pptx/slides/1/
 
 {{< /tabs >}}
 ## **SDKs**
-Using an SDK (API client) is the quickest way for a developer to speed up the development. An SDK takes care of a lot of low-level details of making requests and handling responses and lets you focus on writing code specific to your particular project. Check out our [GitHub repository](https://github.com/aspose-slides-cloud) for a complete list of Aspose.Slides Cloud SDKs along with�working examples, to get you�started in no time. Please check [Available SDKs](/slides/available-sdks/) article to learn how to add an SDK to your project.
+Using an SDK (API client) is the quickest way for a developer to speed up the development. An SDK takes care of a lot of low-level details of making requests and handling responses and lets you focus on writing code specific to your particular project. Check out our [GitHub repository](https://github.com/aspose-slides-cloud) for a complete list of Aspose.Slides Cloud SDKs along with�working examples, to get you�started in no time. Please check [Available SDKs](/slides/available-sdks/) article to learn how to add an SDK to your project.
```

Comment:
pay attention to the spaces  near "working examples, to get you"
Are those special characters?

Reply:
Thank you for the note. Corrected.